src/js/tracks/text-track.js

```diff
@@ -262,9 +214,7 @@ class TextTrack extends EventTarget {
       tt.loaded_ = true;
     }
 
-    if (browser.IS_IE8) {
-      return tt;
-    }
+    return retval;
```

Comment:
We should set `retval` to the response from super or `this` if it isn't in IE8 like it works in the main `track.js` file. Also, a more meaningful name than `retval` would be good.